### PR TITLE
Fixes some issues with the UART connection on GHST TX as well as 500hz/115200baud 

### PR DESCRIPTION
**Pull request by AlessandroAU**

### CODE REVIEW

**Review by CapnBry — APPROVED** [review]

File: src/lib/CRSF/CRSF.cpp [hunk 1]
```diff
@@ -363,7 +363,16 @@ void ICACHE_RAM_ATTR CRSF::sendSyncPacketToTX() // in values in us.
     uint32_t now = millis();
     if (CRSF::CRSFstate && now >= (OpenTXsyncLastSent + OpenTXsyncPacketInterval))
     {
-        uint32_t packetRate = CRSF::RequestedRCpacketInterval * 10; //convert from us to right format
+        uint32_t packetRate;
+        if (CRSF::UARTcurrentBaud == 115200 && CRSF::RequestedRCpacketInterval == 2000)
+        {
+            packetRate = 40000; //constrain to 250hz max
```
Comment (pkendall64) [inline src/lib/CRSF/CRSF.cpp:369]: This should be 4000 not 40000 if you really want 250hz or the comment should be 25hz

**Comment by CapnBry** [comment]
An dammit he got us! I revoke my approval. (hides) Yeah that's a problem right there.

File: src/lib/CRSF/CRSF.cpp (hunk 1 above)
Comment (AlessandroAU) [reply src/lib/CRSF/CRSF.cpp:369]: It's correct, the opentx units are microseconds *!10

Comment (pkendall64) [reply src/lib/CRSF/CRSF.cpp:369]: Damn, schooled again.
Would it be better to keep the original line pulling the value and doing the *10, then a line after checking the baudrate and packetrate and adjusting. Just for readability.

**Pull request merged**